partials/mixins/panel.html

```diff
@@ -0,0 +1,5 @@
+<div id="{{toggle}}-panel" class="reveal js-hidden">
+    <div class="panel-indent{{#getErrorClass}}{{toggle}}{{/getErrorClass}}">
```

Comment:
@gavboulton strange - my html has the same structure but renders without the internal border
```html
<div id="online-toggle-text-panel" class="reveal" aria-hidden="false">
    <div class="panel-indent">
        <div id="online-toggle-text-group" class="form-group validation-error">
            <label for="online-toggle-text" class="form-label-bold">
                <span class="error-message" aria-hidden="true">Tell us your COL number</span>
                What is your COL number?
                <span id="online-toggle-text-hint" class="form-hint">A COL number is a Certificate Online number that is sent in the confirmation email when certificates are ordered online. COL numbers typically look like this COL123456/2015</span>
                <span class="visuallyhidden">Tell us your COL number</span>
            </label>
            <input type="text" name="online-toggle-text" id="online-toggle-text" class="form-control invalid-input" aria-required="true" aria-describedby="online-toggle-text-hint" aria-invalid="true">
        </div>
    </div>
</div>